packages/Radio/src/components/Group/Group.js

```diff
@@ -58,7 +58,7 @@ function Group(props) {
             isChecked: checkedIndex === index,
             isDisabled: isDisabled || child.props.isDisabled,
             canDeselect,
-            name,
+            name: child.props.name ? child.props.name : name,
```

Comment:
In results questionnaires the radio input also needs the name value set in order to scrape the value, so if it is present I am passing it onto the child radio component

Reply:
When the radio components are rendered as children of a group component they will all have the same `name` id set on them, which i believe is correct, however if there is a prop.child it will use that instead.

Reply:
`child.props.name ||  name,`